rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -3401,21 +3401,15 @@ namespace rsx
 
 		if (limit)
 		{
-			const u64 time = rsx::uclock() - Emu.GetPauseTime();
+			const u64 time = get_system_time();
```

Comment:
Any reason to not use the TSC clock here if available?

Reply:
It may be inaccurate on some platforms and this code is called once a frame.

Reply:
It really, shouldn't. This was a huge problem before invariant TSC was a thing, so now processors have to guarantee that the TSC rate is fixed if they advertise the invariant TSC bit in the CPUID. Those platforms may need to be blacklisted in get_tsc_freq instead.
The reason I do not like adding system time is that all dependent logic added in future will need to use it and we may end up in the QPC spam contention hell we had before I sanitized RSX of unnecessary calls to that method.
But for this one case I suppose it is ok.